### PR TITLE
[ADD] udes_stock: Add tool to check for multi-location packages/pallets

### PR DESCRIPTION
**Pull request by williamsajr**
User-story: x12
Signed-off-by: Anthony Williams <anthony.williams@unipart.io>

### CODE REVIEW

File: addons/udes_stock/tools/find_multi_location_packages_and_pallets.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+-- Find packages in multiple locations
+SELECT
+    p.package_id,
+    COUNT(location_id)
+FROM (
+    SELECT
+        package_id,
+        location_id
+    FROM
```
Comment (ws-mps) [inline addons/udes_stock/tools/find_multi_location_packages_and_pallets.sql:11]: We can add a WHERE here to filter out quants without package_id, so we won't need it in the having.

Comment (williamsajr) [reply addons/udes_stock/tools/find_multi_location_packages_and_pallets.sql:11]: Changed

File: addons/udes_stock/tools/find_multi_location_packages_and_pallets.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    LEFT JOIN
+        stock_quant_package pallet ON pallet.id = pack.package_id
+    GROUP BY
+        pallet.id,
+        q.location_id
+) AS p
+GROUP BY
+    p.pallet_id
+HAVING
+    p.pallet_id IS NOT NULL AND COUNT(location_id) > 1
```
Comment (peterpratt13) [inline addons/udes_stock/tools/find_multi_location_packages_and_pallets.sql:45]: I guess miquel wanted this in the where as well?

Comment (peterpratt13) [reply addons/udes_stock/tools/find_multi_location_packages_and_pallets.sql:45]: actually can pallet.id be null?

Comment (williamsajr) [reply addons/udes_stock/tools/find_multi_location_packages_and_pallets.sql]: I've moved the where inside the select

Comment (williamsajr) [reply addons/udes_stock/tools/find_multi_location_packages_and_pallets.sql]: Changed my mind and used the inner join. Good call

**Review by ws-mps — APPROVED** [review]

**Review by peterpratt13 — APPROVED** [review]

**Pull request merged**